packages/oui-modal/_mixins.less

```diff
@@ -92,6 +92,21 @@
       }
     }
 
+    &__loader {
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      margin: -(rem-calc(30)) -(rem-calc(40));
+      background-color: rgba(255, 255, 255, 0.5);
+
+      .oui-spinner {
```

Comment:
I saw that in other branches, you are using flex in this project.

Should we use `flex` instead of `transform` to center the element ?

Reply:
of course, done!